steps/save-carthage-cache/step-info.yml

```diff
@@ -0,0 +1 @@
+maintainer: community
```

Comment:
I have checked the other cache steps in the steplib (save-cocoapods-cache, save-gradle-cache, ...) and all of them have bitrise as the maintainer. Why is this one a community step?

Reply:
Uh oh, no idea. I'll edit this manually in all PRs.